unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/EntityInformation/SmartItems/Parameters/Actions/ActionEventAdapter.cs

```diff
@@ -0,0 +1,100 @@
+using DCL;
+using DCL.Components;
+using DCL.Models;
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class ActionEventAdapter : MonoBehaviour
```

Comment:
I think it would be better to rename or add a namespace making it clear that we are dealing with smart item actions